package.json

```diff
@@ -26,7 +26,6 @@
     "eslint-config-airbnb": "^19.0.4",
     "eslint-config-next": "12.3.1",
     "jest": "^29.1.2",
-    "jest-environment-jsdom": "^29.1.2",
```

Comment:
Can I ask why you deleted this? If we don't need it, sweet, but I found I still needed it to get my tests to pass. I have been doing something funky though, so I'd just love to know why we don't need anymore!

Reply:
Weird that it was deleted, that was not on purpose. For sure will add it back in!

Reply:
If you please go ahead and add that as a required in your review I'll do a fix up, then you can approve (I'll ping you via DM once it's updated)

Reply:
I would not expect any `package.json` or `package-lock.json` changes to exist for this PR, so we need to get them matched up with `main` so they don't appear in the "Files changed list".

Reply:
This isn't the package.json still, so when I run npm run test I get a validation error. I think npm install --save-dev jest-environment-jsdom should do it.